meeting-notes/2018/201Y8-11-12--go-core-dev-team-weekly.md

```diff
@@ -0,0 +1,206 @@
+# 💫Golang Core Dev Team Weekly Sync 🙌🏽 November 12, 2018
+- **Lead:** @eingenito
+- **Notetaker:**
+- **Attendees:**
+  - _attendee names..._
+  - @stebalien
+  - @hannahhoward
+  - @michaelavila
+  - @alanshaw
+  - @warpfork
+  - @momack2
+  - @daviddias
+  
+- **Recording:** _add link to recording once it's online_
+
+## Agenda
+
+- Ask everyone to put their name into the list of attendees
+- Round of updates
+  - What have you accomplished since the last Weekly?
+  - Were there any blockers? If so, which ones? Is it still blocked? Why?
+  - What is the next important thing you should focus on?
+- Ask for general questions. Could be things like:
+  - I'm stuck with something, I don't know who to ask. Who knows who to ask?
+  - Who can help me with xyz?
+- Plan this week
+  - Select issues to work on
+- Review remaining issues if there is time left
+  - Mid-Q OKRs
+
+
+## Notes
+
+### Week Update
+
+@name
+  - Done:
+  - Blocked:
+  - Next:
+
+@marten-seemann
+ - Done:
+ 	- attended IETF 103 meeting & hackathon in Bangkok
+  	- I discovered a packet injection attack on the QUIC handshake, held a presentation about this in the QUIC WG meeting and proposed a fix, which was accepted
+    - discussed loss recovery pitfalls, which will lead to changes to the QUIC specificiation
+ - Blocked: nothing
+ - Next:
+ 	- there was a presentation about using HTTP/2 as a transport. I will follow the discussion about this (maybe this offers an easy solution for our stream muxer problems?)
+ - continue implementing the current QUIC draft
+
+@kevina (might not be able to attend)
+  - Done:
+    - Review p.r. to switch to new commands library.  Almost all the commands are now using the new commands library except for `ipfs ls`.
+  - Next:
+    - Redo `--cid-base` p.r. now that the command line conversion is in, may make things simpler
+
+@schomatis (not attending)
+ - Done/In Progress:
+   - Review `ipfs/docs` PRs (the repo hasn't been active for some months now).
+   - Write content for a concept document about files in IPFS (https://github.com/ipfs/docs/issues/133).
+   - Merge DAG walker (https://github.com/ipfs/go-ipld-format/pull/39).
+   - Review HAMT directory refactor PR (https://github.com/ipfs/go-unixfs/pull/42).
+   - Review trickle DAG refactor PR (https://github.com/ipfs/go-unixfs/pull/10).
+ - Next (not necessarily this week):
+   - With the DAG walker merged finish up the DAG reader refactor using it (https://github.com/ipfs/go-unixfs/pull/12).
+   - Write mid-level document on the `ipfs files` API (https://github.com/ipfs/go-ipfs/pull/5052)
+   - MFS interface (https://github.com/ipfs/go-mfs/issues/3).
+
+@magik6k (not attending)
+ - Done:
+   - Running datastore benchmarks right now
+   - (Almost) have consensus on go-ipfs-files refactor
```

Comment:
What is the `go-ipfs-files` refactor? Any issue/repo to review?

Reply:
https://github.com/ipfs/go-ipfs-files/pull/2